apps/web/.env.local.example

```
@@ -1,4 +1,4 @@
-# Supabase Cloud — get from app.supabase.com → your project → Settings → API
+Supabase Cloud — get from app.supabase.com → your project → Settings → API
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The Supabase Cloud — get from app.supabase.com → your project → Settings → API key has incorrect delimiter

(IncorrectDelimiter)